app.py

```diff
@@ -33,8 +34,11 @@ def create_app() -> Flask:
     flask_app.db = FakeDB()
 
     flask_app.config["SCHEMAS"] = {"towns_fund": parse_schema(deepcopy(TF_SCHEMA))}
-    flask_app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///:memory:"
+    flask_app.config["SQLALCHEMY_DATABASE_URI"] = (
+        "sqlite:///temp.db" if "PERSIST_DB" in os.environ else "sqlite:///:memory:"
```

Comment:
I'll test this but I don't see why it wouldn't work

Reply:
Personally, I only wanted the file-based method so I could connect a watcher

Reply:
nah, it's fine, I was just wondering really.

Reply:
After testing I've discovered it doesn't actually work ! Nice spot. It complains about the db being 'read-only' when file-based. I'm not sure I want to sink time into fixing / working around this in this PR as it blocks my other one - what do you think? I can create a ticket in the backlog so we have reference to it. It'll all change later anyway with Postgres and I don't use Docker when developing really. Do you think you'd use Docker + file-based?

Reply:
It's more for the AWS deployment really. I have used docker locally, but I found it adds nothing ATM as you have to rebuild the image when you make changes, so it's easier to dev locally without it.

However, yeah, not sure we need to sink time in, since this is not going to be the end solution. Maybe @gidsg might have some thoughts?

Reply:
Have you tried it on the deployed AWS instance btw?

Reply:
The AWS instance will still use the default in memory option I assume so that won't change